tests/projects/test_projects.py

```diff
@@ -285,11 +285,12 @@ def test_run_async():
         ),
     ],
 )
-def test_conda_path(mock_env, expected_conda, expected_activate):
+def test_conda_path(mock_env, expected_conda, expected_activate, monkeypatch):
     """Verify that we correctly determine the path to conda executables"""
-    with mock.patch.dict("os.environ", mock_env, clear=True):
-        assert mlflow.utils.conda.get_conda_bin_executable("conda") == expected_conda
-        assert mlflow.utils.conda.get_conda_bin_executable("activate") == expected_activate
+    for k, v in mock_env.items():
+        monkeypatch.setenv(k, v)
```

Comment:
```suggestion
    monkeypatch.setenvs(mock_env)
```

we have provided another method to patch multiple envs

Reply:
Thank you)